x/protorev/keeper/routes.go

```diff
@@ -9,20 +9,9 @@ import (
 	"github.com/osmosis-labs/osmosis/v13/x/protorev/types"
 )
 
-type TradeInfo struct {
-	InputDenom  string
-	OutputDenom string
-	SwapFee     sdk.Dec
-	Pool        gammtypes.CFMMPoolI
-}
-
-type Route struct {
-	Trades []TradeInfo
-}
-
```

Comment:
Per description in the PR, we deprecated the use of this struct in favor of `gammtypes.SwapAmountInRoutes` because we can directly plug this type into the gamm trading interface w/o additional code. As such, all routing logic is now plugging directly into the gammtype which reduces code complexity and improves general design. Outside of that, no logical changes were made to routes.go